src/blocks/index.tsx

```diff
@@ -8,28 +8,44 @@ const DefaultHtmlBlock = loadable(() => import('./format/html'))
 
 export function GetTheBlock(name: string) {
   switch (name) {
-    case 'core/audio': return CommonBlocks.Audio
```

Comment:
I think I much rather prefer the single one line instead of two lines. Makes it easier on the eyes to scan and read

Reply:
Same, but I think it's [not possible](https://github.com/prettier/prettier/issues/6123) with prettier. One workaround is to add this file to prettierignore, but then we lose the benefit of prettier in this file. I'll do as you wish. 

Thanks.

Reply:
Ok no probem. Leave as is. I'll merge now but would be nice if the semicolon issue could be fixed

Reply:
I'll try to fix that in my next pr where I fixed a warning about missing keys on column layouts. Thanks for the merge. Cheers!